HHS/Head-Start-TTADP
# [TTAHUB-836] Print view of goals on goals & objectives tab

## Description of change

View a printable version of the info on the goals and objectives tab. As per group discussion, we are starting with an "all or nothing" approach.

## How to test

View a recipient's goals, click "preview and print." Use the print to PDF button to generate a print preview and see how it looks.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-836

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
